hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/WriteFileThroughputBenchmark.java

```diff
@@ -0,0 +1,232 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership.  The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.hadoop.ozone.freon;
+
+import com.codahale.metrics.Timer;
+import org.apache.hadoop.fs.FSDataOutputStream;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hdds.cli.HddsVersionProvider;
+import org.apache.hadoop.hdds.conf.OzoneConfiguration;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+import picocli.CommandLine.Option;
+import org.slf4j.Logger;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.net.URI;
+import java.util.concurrent.Callable;
+
+/**
+ * File Generator for populating the cluster with random files.
+ */
+@CommandLine.Command(name = "wtb",
+    aliases = "write-throughput-benchmark",
+    description = "Benchmark for creating a file",
+    versionProvider = HddsVersionProvider.class,
+    mixinStandardHelpOptions = true,
+    showDefaultValues = true)
+public class WriteFileThroughputBenchmark extends BaseFreonGenerator
+    implements Callable<Void> {
+
+  @Option(names = {"-p", "--path"},
+      description = "Ozone filesystem path OFS scheme",
+      defaultValue = "ofs://ozone1/volume1/bucket1")
+  private String rootPath;
+
+  @Option(names = {"-s", "--size"},
+      description = "Size of each generated files (in GB)",
+      defaultValue = "1")
+  private long fileSize;
+
+  @Option(names = {"-bl", "--block"},
+      description = "Specify the Block Size in MB",
+      defaultValue = "128")
+  private long blockSize;
```

Comment:
I don't think Ozone supports custom block size for each file.

Reply:
Since this patch could be used for different  Apache Hadoop compatible filesystems (e.g. HDFS, Ozone, S3A, WASB), hence feature would be applicable for those filesystems that support custom block size.